csharp-refactoring/RefactoringObjectOrientationAbusers/RefactoringObjectOrientationAbusers/TemporaryField/Correct/BookstoreCustomer.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace RefactoringObjectOrientationAbusers.TemporaryField.Correct
+{
+    public class BookstoreCustomer
+    {
+        public ICollection<string> BoughtBooks { get; set; }
+
+        public double GetDiscountRate()
+        {
+            var boughtBooksCount = BoughtBooks.Count;
+            var discountTreshold = GetDiscountThreshold(boughtBooksCount);
```

Comment:
Should be Threshold, not Treshold. 
Please look at all occurences of Treshold. 
And fix the code snippets in the article as well